### PR TITLE
Move style source selections to build

### PR DESCRIPTION
**Pull request by TrySound**
We want to have freedom in manipulation and reusability of tree parts so gonna merge tree data of all pages into build.

Here migrated style source selections. Left one todo to cleanup tree data of deleted pages. Though easier to solve this when pages management is moved to client.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - test it on preview
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by kof — APPROVED** [review]

File: apps/builder/app/routes/s/css.ts [hunk 1]
```diff
@@ -47,7 +46,7 @@ export const loader = async ({ request }: ActionArgs) => {
         assets: canvasData.assets,
         breakpoints: canvasData.build?.breakpoints,
         styles: canvasData.build?.styles,
-        styleSourceSelections: canvasData.tree?.styleSourceSelections,
+        styleSourceSelections: canvasData.build?.styleSourceSelections,
```
Comment (istarkov) [inline apps/builder/app/routes/s/css.ts:49]: please add to saas link to this change, we have the same code there.

File: packages/prisma-client/prisma/schema.prisma [hunk 2]
```diff
@@ -81,9 +81,10 @@ model Build {
   isDev  Boolean // exctly one is true per project
   isProd Boolean // at most one is true per project (none if not published)
 
-  breakpoints  String @default("[]")
-  styles       String @default("[]")
-  styleSources String @default("[]")
+  breakpoints           String @default("[]")
+  styles                String @default("[]")
+  styleSources          String @default("[]")
+  styleSourceSelections String @default("[]")
```
Comment (istarkov) [inline packages/prisma-client/prisma/schema.prisma:87]: Next PR should be removal of styleSelections from tree?

Comment (istarkov) [reply packages/prisma-client/prisma/schema.prisma:87]: > Next PR should be removal of styleSelections from tree?

plz add an issue about outdated fields removal

Comment (TrySound) [reply packages/prisma-client/prisma/schema.prisma:87]: We move all data from Tree table and remove it completely. So selective removal not necessary.

File: apps/builder/app/routes/s/css.ts (hunk 1 above)
Comment (TrySound) [reply apps/builder/app/routes/s/css.ts:49]: Created an issue

**Pull request merged**